myhpom/admin.py

```diff
@@ -30,21 +32,64 @@ class StateAdmin(admin.ModelAdmin):
 
 class DocumentUrlInlineAdmin(admin.StackedInline):
     model = DocumentUrl
-    fields = ('url_as_link', 'expiration',)
-    readonly_fields = ('url_as_link',)
+    fields = ('url_as_link', 'expiration', 'cloud_factory_runs')
+    readonly_fields = ('url_as_link', 'cloud_factory_runs')
     extra = 0
 
     def url_as_link(self, obj):
         return mark_safe(
-            '<a href="%s" target="_blank" rel="noopener noreferrer">%s</a>'
-            % (obj.url, obj.url))
+            '<a href="%s" target="_blank" rel="noopener noreferrer">%s</a>' % (obj.url, obj.url)
+        )
+
+    def cloud_factory_runs(self, obj):
+        return mark_safe(
+            '<ul>'
+            + ''.join(
+                # Is there a better way to get a URL in the mmh-admin? I don't find a resolver.
+                '<li><a href="/mmh-admin/myhpom/cloudfactorydocumentrun/%d">%s</a></li>'
```

Comment:
oh yeah, that is always a pain, finding these admin resolvers. I like django-extensions for that...`python manage.py show_urls` will then tell you that this url is...`admin:myhpom_cloudfactorydocumentrun_change`

Reply:
Getting W503 from flake8  here (don't put an operator at the beginning of a line)

Reply:
Oh, flake8, right.